setup.cfg

```diff
@@ -35,7 +35,7 @@ setup_requires = setuptools_scm
 
 install_requires =
 	einops
-  imageio>=2.5.0,!=2.11.0,!=2.22.1
+    imageio>=2.5.0,!=2.11.0,!=2.22.1
```

Comment:
does this just look funny because of a rendering thing or is it actually out of line?

Reply:
Good eye! The spacing was actually weird. I've fixed it.